Core/Helper.php

```diff
@@ -366,4 +363,19 @@ public static function getShop()
         $oShop->load($sShopId);
         return $oShop;
     }
+
+    /**
+     * Generates a unique token
+     *
+     * @return string
+     *
+     * @since 1.2.0
+     */
+    public static function getUniqueToken()
```

Comment:
`OxidEsales\EshopCommunity\Core\UniversallyUniqueIdGenerator` has some methods to create unique tokens built in.

Reply:
that token is for ratepay and has to have the session id